dockers/docker-orchagent/supervisord.conf.j2

```diff
@@ -28,49 +28,69 @@ stdout_logfile=syslog
 stderr_logfile=syslog
 dependent_startup=true
 
-[program:gearsyncd]
-command=/usr/bin/gearsyncd -p /usr/share/sonic/hwsku/gearbox_config.json
-priority=3
+{% set is_fabric_asic = 0 %}
+{% set orchagent_dependent_startup_wait_for = "portsyncd:running" %}
+{% if DEVICE_METADATA.localhost.switch_type %}
+{% if DEVICE_METADATA.localhost.switch_type == "fabric" %}
+{% set is_fabric_asic = 1 %}
+{% set orchagent_dependent_startup_wait_for = "rsyslogd:running" %}
+{%- endif %}
+{%- endif %}
+
+[program:orchagent]
```

Comment:
My concern is, priorities are all jumbled up here. IMO, its harder to add a process. If you dont prefer to have duplicate, please have it in the order of priority and add the skip the ones for fabric within "if" case

Reply:
Sorting based on priority makes sense.

For clarification, I will add the processes which do NOT run on fabric in "if" case.